### DISCUSSION TITLE
Is Postgres still on the planing?

### DISCUSSION THREAD

**Discussion by BobWs** [mdouchement/standardfile#95] [General]
Hi,
Is support for Postgres database still on the planning? Or even Mariadb would be fine!
TIA

**Comment by mdouchement** [comment]
Hello,

Postgres is way better than MySQL/MariaDB.
Currently the embedded database is enough for a personnal usage with several users, I didn't see any limitations.

**Reply by binaryben** [reply]
Postgres would be useful for HA deployments (host one instance on a home lab server and another in the cloud). Does anything special need to be done to backup the embedded database? SQLite needs the connection to be stopped first for example.

**Reply by mdouchement** [reply]
I use `rclone` to do backup of the database after stopping temporarily the application.

I run a `cp` to copy the database while the application is running, the checksum of both copies was the same, so it should not be an issue. That said, extensive tests should be executed to ensure there is no issue.

**Reply by binaryben** [reply]
Thanks @mdouchement, should be pretty easy to add to Kopia then. Postgres would still be very handy if you ever have the time and motivation 😊 Unfortunately I'm not well versed in Go so can't give a PR at the moment.
